.pre-commit-config.yaml

```diff
@@ -1,30 +1,13 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 24.3.0
-    hooks:
-      - id: black
-        args: ["--target-version", "py38"]
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.13.2
-    hooks:
-      - id: isort
-        args: ["--profile", "black"]
   - repo: https://github.com/pre-commit/mirrors-prettier
     rev: v3.1.0
     hooks:
       - id: prettier
-        args: [--prose-wrap=preserve, --print-width=90]
-  - repo: https://github.com/pycqa/flake8
-    rev: 7.0.0
-    hooks:
-      - id: flake8
+        args: [--prose-wrap=preserve, --print-width=88]
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: v0.3.7
     hooks:
       - id: ruff
+        args:
+          - --fix
       - id: ruff-format
-  - repo: https://github.com/asottile/pyupgrade
```

Comment:
@andhus  IMHO i'd keep this hook in, it forces contributors to not use old way of coding that was replaced with something more efficient / readable in latter python versions ("".format() > f"", etc)